apps-rendering/webpack.config.ts

```diff
@@ -109,6 +109,26 @@ const serverConfig = (
 						},
 					],
 				},
+				{
+					// @guardian packages must be transpiled
+					// https://github.com/guardian/recommendations/blob/main/npm-packages.md#using-guardian-npm-packages
+					test: /@guardian\/.+\.js$/,
```

Comment:
We need to add `@guardian` libraries to the babel pipeline in AR, otherwise the server won't start